koku/providers/gcp/provider.py

```diff
@@ -40,6 +40,13 @@ def cost_usage_source_is_reachable(self, credential_name, data_source):
             key = 'billing_source.bucket'
             raise serializers.ValidationError(error_obj(key, e.message))
 
+        report_prefix = data_source.get('report_prefix', '')
+        if report_prefix:
+            if len(report_prefix) > 64:
```

Comment:
For easier future revision, let's put this `64` in a module-level or class-level variable and also use it in `test_bucket_report_prefix_too_long` for its too-long value.